lib/annotate/annotate_models.rb

```diff
@@ -290,11 +290,14 @@ def get_schema_info(klass, header, options = {})
             end
           end
         end
+
+        # Generate the column name optionally including column comments
         col_name = if with_comments?(klass, options) && col.comment
-                     "#{col.name}(#{col.comment})"
+                     "#{col.name}(#{col.comment.gsub(/\n/, "\\n")})"
```

Comment:
I wanted to find something more general and elegant than `gsub`, but didn't find a good option. I'd be interested if there are other characters that would be concerning (`\t`?) that would make sense to check for and escape as well.

Reply:
Maybe there's a library for sanitizing whitespace? It might be worth looking into.

Reply:
I looked a bit, but didn't find anything that stuck out. [`String#dump`](https://ruby-doc.org/core-2.6/String.html#method-i-dump) seemed promising, but I believe it escaped UTF characters that were expected to work.

I could make this more generic and escape any `\` with `\\` if that seems safer.